Makefile

```diff
@@ -44,14 +46,19 @@ generate:
 # Build the docker image
 docker-build: resolve test
 	docker build . -t ${IMG}
+	docker tag ${IMG} ${IMG-CI}
 	@echo "updating kustomize image patch file for manager resource"
-	sed -i'' -e 's@image: .*@image: '"${IMG}"'@' ./config/default/manager_image_patch.yaml
+	sed -i'' -e 's@image: .*@image: '"${IMG-CI}"'@' ./config/default/manager_image_patch.yaml
 
 # Push the docker image
 docker-push:
-	docker push ${IMG}
```

Comment:
now docker push is empty. Is it on purpose?

Reply:
nope, it was for testing purpose, reverted